Makefile

```diff
@@ -0,0 +1,53 @@
+.PHONE: vm-kernel
+vm-kernel:
+	cd neonvm && make kernel
```

Comment:
```make -C neonvm kernel``` does this without the `cd`

similarly it'd be fun if `neonvm` were a target here that did submodule init or something related.

Reply:
> `make -C neonvm kernel` does this without the `cd`

Nice! Reusing this.

> similarly it'd be fun if neonvm were a target here that did submodule init or something related.

Will do something like this in the following work with Makefile